assets/css/smallscreen.css

```diff
@@ -1,55 +1,128 @@
-/* =============================For Tablet View========================== */
-@media screen and (min-device-width: 604px) {
-  /* and  (max-device-width: 1024px) */
-  header {
-    height: 5.7%;
-    width: 100%;
-    position: fixed;
-    top: 0;
-  }
+header {
+  flex-direction: row;
+  display: flex;
+  justify-content: space-between;
+  align-items: start;
+  height: 5.7%;
+  width: 100%;
+  position: fixed;
+  padding-bottom: 15px;
+  background-color: #3c3a39;
+  top: 0;
+}
 
-  header nav {
-    background-color: #3c3a39;
-    padding-bottom: 15px;
-    width: 100%;
-    position: fixed;
-    top: 0;
-    flex-direction: row;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-  }
+nav {
+  padding-right: 24px;
+}
+
+.brand,
+.hamburger-mobile-icon {
+  padding-top: 20px;
+}
+
+.logo {
+  padding-left: 24px;
+  font-weight: bold;
+  letter-spacing: 2px;
+  font-size: 18px;
+  color: #fff;
+  line-height: 24px;
+  text-decoration-line: none;
+}
+
+.dark-logo {
+  color: #3c3a39;
+}
+
+/* Mobile menu css */
+.hamburger-mobile-icon {
+  display: none;
+  margin: auto 0;
+  cursor: pointer;
+}
+
+.hamburger-mobile-icon div {
+  width: 17px;
+  height: 3px;
+  background-color: #acb7c3;
+  margin: 3px 0;
+  transition: 0.4s;
+}
+
+.open .bar1 {
+  -webkit-transform: rotate(-45deg) translate(-6px, 6px);
+  transform: rotate(-45deg) translate(-6px, 6px);
+  width: 20px;
+  margin: 6px 0;
+}
+
+.open .bar2 {
+  opacity: 0;
+}
+
+.open .bar3 {
+  -webkit-transform: rotate(45deg) translate(-6px, -8px);
+  transform: rotate(45deg) translate(-6px, -8px);
+  width: 20px;
+  margin: 6px 0;
+}
+
+.mobile-icon-menu {
+  display: none;
+  position: absolute;
+  background-color: #3c3a39;
+  top: 40px;
+  left: 0;
+  height: calc(95vh - 50px);
+  width: 100%;
+}
```

Comment:
- [ ] Kindly play with your top property to adjust this spacing above the hamburger menu.
- [ ] Consider using the same font and colors as given in the design, please take a look back to the Figma design and rectify your design with correct data 😄 
- [ YOU] 
<img width="525" alt="font" src="https://user-images.githubusercontent.com/79058364/157254321-7c17736d-a034-4e3f-882b-786acf412fd4.png">


- [ FIGMA]

<img width="394" alt="menu" src="https://user-images.githubusercontent.com/79058364/157253908-190f914a-0d54-44ae-bf4f-b7dff6289486.png">
`